deploy/charts/harvester-node-disk-manager/templates/crds/harvesterhci.io_lvmvolumegroups.yaml

```
@@ -0,0 +1,133 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    {}
+  name: lvmvolumegroups.harvesterhci.io
+spec:
+  group: harvesterhci.io
+  names:
+    kind: LVMVolumeGroup
+    listKind: LVMVolumeGroupList
+    plural: lvmvolumegroups
+    shortNames:
+    - lvmvg
+    - lvmvgs
+    singular: lvmvolumegroup
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.parameters
+      name: Parameters
+      type: string
+    - jsonPath: .status.vgStatus
+      name: Status
+      type: string
+    - jsonPath: .spec.nodeName
+      name: Node
+      type: string
+    name: v1beta1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              desireState:
```

I think `desiredState` is a more correct english. @tserong What do you say to this as a native English speaker?
```suggestion
              desiredState:
```

---

I thought you are correct, updated!

---

Yep, agreed

---

It looks like this has been updated to `DesiredState` in the description, but the field names are still `desireState`

---

yaml field will start with lower case. You can also check the following nodeName. It's the same case.

---

Sorry, I mean the field names say `desire` as opposed to `desired` (trailing `d`)

---

nice catch... I missed that one

---

updated!